esmvalcore/_recipe.py

```diff
@@ -361,6 +362,45 @@ def _get_default_settings(variable, config_user, derive=False):
     return settings
 
 
+def get_input_fx_filelist(variable, rootpath, drs):
```

Comment:
I think this function should be removed altogether, it looks like it migrated here from _data_finder.py

Reply:
reply below in the main body of the issue